tests/server.js

```diff
@@ -42,11 +44,42 @@ const userCheck = (usr) => {
   });
 };
 
+const service = [
+  {
+    id: serviceUuid,
+    language: 'us-en',
+    publishable: false,
+    value: {},
+  },
+];
+
+const getService = (svc) => {
+  return database.select('*').from('content-type--services').where({
+    id: svc.id,
+  });
+};
+
+const serviceCheck = (svc) => {
+  return getService(svc).then(srvc => {
+    if (srvc.length < 1) {
+      return database('content-type--services').insert(svc);
+    }
+
+    return [];
+  });
+};
+
+const deleteService = () => {
+  return database('content-type--services').where('id', '=', serviceUuid).del();
+};
+
 // cue up our application for all tests
 test.cb.before((t) => {
   punchcard().then(app => {
     agent = request.agent(app);
 
+    return Promise.map(service, serviceCheck);
```

Comment:
@Snugug, @Snugug said it is ok to not move this function (serviceCheck) into `before` and to disregard the original comment by @Snugug